README.md

```diff
@@ -121,17 +121,100 @@ Resolved environments can also be created via the API:
 
 ## Quickstart
 
-First, install Rez. Download the source, and from the source directory, run
-(with DEST_DIR replaced with your install location):
+Choose your installation method.
 
-    ]$ python ./install.py -v DEST_DIR
+<details>
+  <summary>Simple</summary>
+<table>
+<tr><td>
+<br>
+
+**Simple** The simple approach is well suited for beginners and those looking to learn more about Rez and whether it is suited to their problem and environment.
+
+</td></tr>
+<tr><td>
+
+```bash
+$ pip install rez
```

Comment:
We shouldn't advertise this method first. We should advertise to install in a virtualenv or install with `pip install rez --user`, but not install on the system. Installing on the system is never recommended and the python community puts a lot of effort to emphasis on the fact that we should never alter the system's python and always install with `--user` or use virtualenvs.

Why should we never alter the system's python site-packages? Because for platforms that comes with python already installed and that the OS is using python to work properly (Linux and OSX), it's almost impossible (if not completely impossible) to revert to the initial state if something goes wrong without re-installing the OS.